api/api.go

```diff
@@ -162,6 +162,7 @@ func New(requiredUserAgent string, requiredPassword string, cs modules.Consensus
 	if api.cs != nil {
 		router.GET("/consensus", api.consensusHandler)
 		router.POST("/consensus/validate/transactionset", api.consensusValidateTransactionsetHandler)
+		router.GET("/consensus/blocks/:height", api.consensusBlocksHandler)
```

Comment:
since at zero point we dont have any hash - easier way to get first data is just height of block.

Optionally ofc can be queried by hash, but i see no need for me.